packages/test/src/test-integration-update.ts

```diff
@@ -45,6 +55,151 @@ export async function workflowWithUpdates(): Promise<string[]> {
   return state;
 }
 
+test('UWS happy path', async (t) => {
```

Comment:
Please expand the `UWS` abbreviation here and below.

```suggestion
test('UpdateWithStart - Happy path', async (t) => {
```

Reply:
Thanks, done.